docs/source/tutorials/pred_probs_cross_val/pred_probs_cross_val.rst

```diff
@@ -21,7 +21,11 @@ When using ``cleanlab``, we will typically want to find label issues in all labe
 What is K-fold cross-validation?
 --------------------------------
 
-K-fold cross-validation partitions the entire dataset into *K* disjoint subsets of data called **folds**. *K* independent copies of our model are trained, where for each model copy, one fold of the data is held out from its training (the data in this fold may be viewed as a **validation set** for this copy of the model). Each copy of the model has a different validation set for which we can obtain out-of-sample predicted probabilities from this copy of the model. Since each datapoint is held-out from one copy of the model, this process allows us to get out-of-sample predictions for every datapoint! We, in particular, recommend applying stratified cross-validation, which tries to ensure the proportions of data from each class match across different folds.
+.. image:: pred_probs_cross_val.png
```

Comment:
Good call on the image size. Maybe let's move it here?

https://github.com/cleanlab/assets

Reply:
moved image to here: https://github.com/cleanlab/assets/blob/master/cleanlab/pred_probs_cross_val.png